comptests/prov/test_prov_k8s_cortx.py

```diff
@@ -28,7 +28,7 @@
 from commons import commands
 from commons.helpers.pods_helper import LogicalNode
 from commons.utils import assert_utils
-from config import CMN_CFG, PROV_CFG
+from config import CMN_CFG, PROV_CFG, PROV_TEST_CFG
```

Comment:
Make this import multi line.

Reply:
1. As sanity is running in my local branch so only pointing to local branch. Before merging the PR changes will be made accordingly.
2. Done.